presto-main/src/test/java/com/facebook/presto/execution/TestQueryStats.java

```diff
@@ -58,16 +58,17 @@
                     succinctBytes(116L),
                     117L,
                     succinctBytes(118L),
-                    new Duration(119, NANOSECONDS),
-                    120L,
-                    new Duration(121, NANOSECONDS),
+                    new Duration(120, NANOSECONDS),
```

Comment:
This change seems unrelated. Please just add the new line

Reply:
Done.  Please review.